### DISCUSSION TITLE
Enable/Disable features in Cargo build

### DISCUSSION THREAD

**Discussion by ramenhost** [zed-industries/zed#15550] [Feature Requests]
It would be nice to have an option to disable features during compilation. For example, I am looking for a way to build zed without collaboration features for an enterprise setting.

Preferably using `--features` option of `cargo build`.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
